per/views.py

```diff
@@ -429,6 +429,34 @@ def handle_post(self, request, *args, **kwargs):
 
         return JsonResponse({'status': 'ok'})
 
+def delete_draft(raw):
```

Comment:
Can we get this function to accept the `id` directly?

Reply:
Would this suffice the change you want: https://github.com/IFRCGo/go-api/commit/d7be6afd3bdfdc012c65e126fc7e991c04bcbeaa ?